game.py

```diff
@@ -61,35 +74,27 @@ def create_code_word(self):
     def take_guess(self):
         '''The decoder takes their guess in this function'''
         # input() is used to pause the game before it continues to the next segment here. The prompt is in board.render
```

Comment:
You could consider moving the call to `input()` to the render function to remove the need to explain yourself with a comment.